decision_service/include/data.h

```diff
@@ -0,0 +1,389 @@
+
+unsigned char regression_data_1_model[] = {
```

Comment:
How is this file created?

Reply:
will be created by vwslim. working on another pull request.

---
In reply to: [192291382](https://github.com/JohnLangford/vowpal_wabbit/pull/1492#discussion_r192291382) [](ancestors = 192291382)

Reply:
Ok.  I am guessing you will remove this file after that since it will be generated?

---
In reply to: [192383888](https://github.com/JohnLangford/vowpal_wabbit/pull/1492#discussion_r192383888) [](ancestors = 192383888,192291382)